server/src/main/java/org/apache/uniffle/server/ShuffleTaskManager.java

```diff
@@ -771,7 +772,9 @@ public void removeResources(String appId, boolean checkAppExpired) {
       partitionsToBlockIds.remove(appId);
       shuffleBufferManager.removeBuffer(appId);
       shuffleFlushManager.removeResources(appId);
-      storageManager.removeResources(new AppPurgeEvent(appId, shuffleTaskInfo.getUser()));
+      storageManager.removeResources(
+          new AppPurgeEvent(
```

Comment:
Should we remove the following constructor? For prevent the same problem from happening again.
https://github.com/apache/incubator-uniffle/blob/917456c5eb0d7b70f7faa5da489aaa29d54cde40/server/src/main/java/org/apache/uniffle/server/event/AppPurgeEvent.java#L28-L30

Reply:
This will bring too many changes.